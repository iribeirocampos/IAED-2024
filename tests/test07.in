p parque1 10 0.30 0.50 15.00
p
e parque1 AA-00-AA 01-01-2024 8:00
e parque1 AA-01-AA 01-01-2024 8:01
e parque1 AA-02-AA 01-01-2024 8:02
e parque1 AA-03-AA 01-01-2024 8:03
e parque1 AA-04-AA 01-01-2024 8:04
e parque1 AA-05-AA 01-01-2024 8:05
e parque1 AA-06-AA 01-01-2024 8:06
e parque1 AA-07-AA 01-01-2024 8:07
e parque1 AA-08-AA 01-01-2024 8:08
e parque1 AA-09-AA 01-01-2024 8:09
p
s parque1 AA-00-AA 01-01-2024 9:59
s parque1 AA-08-AA 01-01-2024 10:00
s parque1 AA-05-AA 01-01-2024 10:20
s parque1 AA-02-AA 01-01-2024 11:01
p
q
