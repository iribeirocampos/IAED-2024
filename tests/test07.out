parque1 10 10
parque1 9
parque1 8
parque1 7
parque1 6
parque1 5
parque1 4
parque1 3
parque1 2
parque1 1
parque1 0
parque1 10 0
AA-00-AA 01-01-2024 08:00 01-01-2024 09:59 3.20
AA-08-AA 01-01-2024 08:08 01-01-2024 10:00 3.20
AA-05-AA 01-01-2024 08:05 01-01-2024 10:20 3.70
AA-02-AA 01-01-2024 08:02 01-01-2024 11:01 5.20
parque1 10 4
